resources/js/store/auth/index.js

```diff
@@ -14,6 +22,23 @@ const register = {
     },
   },
   actions: {
+    login(context, userData) {
+      axios
+        .post("/login", {
+          email: userData.email,
+          password: userData.password,
+          remember: userData.remember,
+        })
+        .then(function (response) {
+          //   console.log(response);
+          context.commit("login", response);
+        })
+        .catch(function (error) {
+          //   console.log(error.response.data.errors);
+          context.commit("login", error.response.data);
```

Comment:
If we need to save errors then the mutation name should be `error` and not `login`